manager/src/main/java/org/openremote/manager/app/ConfigurationService.java

```diff
@@ -82,9 +108,9 @@ public void stop(Container container) throws Exception {
     }
 
 
-    public void saveMangerConfig(Object managerConfiguration) {
+    public void saveManagerConfigFile(Object managerConfiguration) {
         LOG.log(Level.INFO, "Saving manager_config.json..");
-        try (OutputStream out = new FileOutputStream(pathPublicRoot + "/manager_config.json")) {
+        try (OutputStream out = new FileOutputStream(persistenceService.getStorageDir().resolve("manager").resolve("manager_config.json").toFile())) {
```

Comment:
Be consistent use a method `getManagerConfigPath()` which is then called by `getManagerConfiguImagePath()` and then both are used in all path references